src/ReactiveUI/Platforms/netstandard1.1/PlatformRegistrations.cs

```diff
@@ -1,16 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MS-PL license.
-// See the LICENSE file in the project root for more information.
-
-using System;
-
-namespace ReactiveUI
-{
-    public class PlatformRegistrations : IWantsToRegisterStuff
-    {
-        public void Register(Action<Func<object>, Type> registerFunction)
-        {
-            throw new Exception("You are referencing the Portable version of ReactiveUI in an App. Reference the platform-specific version.");
```

Comment:
This is a big change that I'm not comfortable with. We used to throw if the end user didn't reference the right packages as part of bait and switch but now we don't. Now it'll be harder for the end user and us to troubleshoot registrations as previously we had certainty that things would explode if the user did something wrong. Now it's silent. Thoughts?

Reply:
I think I'm happy either way TBH.

From what @onovotny has said, PackageRefs should prevent the problem we used to have from happening (e.g. if you import RxUI in a shared project, and then use that in an app directly, you should get the platform specific version rather than the netstandard version). 

I think it's fairly minor either way, so if we're worried I think better to ship with the status quo and then we can merge netcoreapp and netstandard when we're happy the tooling will solve it.

Reply:
You do not need to reference the right packages anymore because NuGet is better. PackageRef does this always. The only people this bits are people who are using packages.config and mess up.

The Xamarin Templates, and all the other defaults everywhere, use PackageRef now.

Reply:
The only situation where this might still go wrong is for VS4Mac users. As adding the first package to PackageRef manually is still needed for it to use PackageRefs. By default it still assumes packages.config. I hope this will be fixed in 7.5.

Even if you set up a new solution, it adds required packages in a packages.config file. Annoying...